rezz.lic

```diff
@@ -1,119 +1,126 @@
+      Flags['no-favor'] ? begin echo "*** #{name} has no favors! ***"; next end : nil
+      rezz_process(name)
+	end
   end
 
-  def rezz_prep(person)
-    # cast buffs if set
-    check_buffs
-
-    # perceive body & rejuv
-    rejuv(person)
-
-    # cast rezz
-    mana = cast_rezz
-
-    # infuse untill see spirit
-    find_soul(person, mana)
-
-    # snap cast soul bonding
-    cast_soulbonding(person)
-
-    # check for silver again. recast rejuv (or infuse again) if not
-    rejuv(person)
-
-    # gesture
-    bput("gest #{person}", 'Roundtime')
+  def find_dead
+    /You take a moment to look for everybody in the area and see (.*)/ =~ bput('look people', "You look around and notice that you're the only one in the area", /You take a moment to look for everybody in the area and see (.*)/)
+    @target = Regexp.last_match(1).sub(/ and /, ', ').scan(/the body of ([a-z[:blank:]]+) who is [a-z[:blank:]]+?,?./i).flatten.map { |obj| obj.split.last }
+	@target == [] ? @target = nil : nil
   end
 
-  def find_soul(person, infuse_amt)
-    loop do
-      pause 5 while mana <= 40
-      harness_mana([infuse_amt]) unless skip_harness?
-      case bput("infuse rezz #{infuse_amt}", /You clearly sense the spirit of #{person} in the Void/i, 'You sense a spirit nearby, but you are unable to make it out clearly.', 'Your sense of sight has not fully transitioned to the spirit world', "You don't have enough harnessed mana to infuse that much.")
-      when 'You sense a spirit nearby, but you are unable to make it out clearly.'
-        next
-      when /You clearly sense the spirit of #{person} in the Void/i
-        break
-      when 'Your sense of sight has not fully transitioned to the spirit world'
-        pause 2
-        next
-      when "You don't have enough harnessed mana to infuse that much."
-        next
-      else
-        next
-      end
+  def rejuv(player, quick = false)
+    Flags.reset('rejuv-silver')
+    quick ? begin @rejuv['cambrinth'] = []; @rejuv['prep_time'] = 5; @rejuv['mana'] = 10 end : nil
+    until Flags['rejuv-silver'] || Flags['alive'] || Flags['necro']
+	  pause 5 while DRStats.mana < 40
+      cast_spell(@rejuv, @settings)
     end
   end
 
-  def is_silver?(person)
-    /As you concentrate, a thin (.*) nimbus/ =~ bput("perc #{person}", 'As you concentrate, a thin (.*) nimbus')
-    color = Regexp.last_match(1)
-    echo "Memories are #{color}"
-    waitrt?
-    color.eql? 'silver'
-  end
-
-  def rejuv(person)
-    return if is_silver?(person)
-    data = get_data('spells').spell_data['Rejuvenation']
-    data['cast'] = "cast #{person}"
-    if @rejuv_mana.zero?
-      data = check_discern(data, @settings)
-      @rejuv_mana = data['mana']
-    end
-
-    loop do
-      cast_spell(data, @settings)
-      break if is_silver?(person)
+  def find_soul(player)
+    10.times do
+	  pause 5 while DRStats.mana < 40
+      @settings.osrel_no_harness ? nil : harness_mana([@rezz['mana']])
```

Comment:
As a side note here, I usually cast rezz at 20 mana, and harness at 15. Usually I've went with around 75% mana on harness to rezz ratio.

Reply:
Why not just prep at 15 too then?  Prepping at 20 in that case just makes it use more mana each time it pulses.

Reply:
@jayRyan24 See previous comment.